### PR TITLE
OP-655 Use interface instead of concreate class

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-655

### CODE REVIEW

File: src/main/java/org/isf/accounting/rest/BillController.java [hunk 1]
```diff
@@ -114,8 +114,8 @@ public BillController(BillBrowserManager billManager, PriceListManager priceList
         Bill bill = billMapper.map2Model(newBillDto.getBill());
         
         Patient pat = patientManager.getPatientByName(bill.getPatName()); //FIXME: verify why we were searching by name
-        
-        ArrayList<PriceList> list = priceListManager.getLists();
+
+		List<PriceList> list = priceListManager.getLists();
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/rest/BillController.java:118]: indentation?

File: src/main/java/org/isf/accounting/rest/BillController.java [hunk 2]
```diff
@@ -165,8 +165,8 @@ public BillController(BillBrowserManager billManager, PriceListManager priceList
         }
         
         Patient pat = patientManager.getPatientByName(bill.getPatName()); //FIXME: verify why we were searching by name
-        
-        ArrayList<PriceList> list = priceListManager.getLists();
+
+		List<PriceList> list = priceListManager.getLists();
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/rest/BillController.java:169]: indentation?

File: src/main/java/org/isf/accounting/rest/BillController.java [hunk 3]
```diff
@@ -215,8 +215,8 @@ public BillController(BillBrowserManager billManager, PriceListManager priceList
         
         GregorianCalendar dateto = new GregorianCalendar();
         dateto.setTime(dateTo);
-        
-        ArrayList<Bill> bills;
+
+		List<Bill> bills;
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/rest/BillController.java:219]: indentation?

File: src/main/java/org/isf/vaccine/rest/VaccineController.java [hunk 4]
```diff
@@ -93,7 +92,7 @@ public VaccineController(VaccineBrowserManager vaccineManager, VaccineMapper vac
     @GetMapping(value = "/vaccines/type-code/{vaccineTypeCode}", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<VaccineDTO>> getVaccinesByVaccineTypeCode(@PathVariable String vaccineTypeCode) throws OHServiceException {
 	    LOGGER.info("Get vaccine by code: {}", vaccineTypeCode);
```
Comment (mwithi) [inline src/main/java/org/isf/vaccine/rest/VaccineController.java:94]: indentation here

File: src/main/java/org/isf/visits/rest/VisitsController.java [hunk 5]
```diff
@@ -73,7 +73,7 @@ public VisitsController(VisitManager visitManager, VisitMapper visitMapper) {
     @GetMapping(value = "/visit/{patID}", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<VisitDTO>> getVisit(@PathVariable int patID) throws OHServiceException {
 	    LOGGER.info("Get visit related to patId: {}", patID);
```
Comment (mwithi) [inline src/main/java/org/isf/visits/rest/VisitsController.java:75]: indentation here

**Review by mwithi — CHANGES_REQUESTED** [review]
Small changes required

File: src/main/java/org/isf/visits/rest/VisitsController.java (hunk 5 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/visits/rest/VisitsController.java]: Fixed

File: src/main/java/org/isf/vaccine/rest/VaccineController.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/vaccine/rest/VaccineController.java]: Fixed

File: src/main/java/org/isf/accounting/rest/BillController.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/rest/BillController.java:219]: Fixed

File: src/main/java/org/isf/accounting/rest/BillController.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/rest/BillController.java]: Fixed

File: src/main/java/org/isf/accounting/rest/BillController.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/rest/BillController.java:118]: Fixed

**Review by mwithi — APPROVED** [review]

**Pull request merged**